kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableConfig.java

```diff
@@ -105,6 +106,61 @@ public class TableConfig<T> {
                     "needs to be a long."
             );
 
+    /*
+     * This table property is used to track the commit-coordinator name for this table. If this
+     * property is not set, the table will be considered as file system table and commits will be
+     * done via atomically publishing the commit file.
+     */
+    public static final TableConfig<Optional<String>> COORDINATED_COMMITS_COORDINATOR_NAME =
+            new TableConfig<>(
+                    "delta.coordinatedCommits.commitCoordinator-preview",
+                    null, /* default values */
+                    Optional::ofNullable,
+                    value -> true,
+                    "The commit-coordinator name for this table. This is used to determine\n" +
```

Comment:
The use of \n doesn't look right? I don't think that this will render correctly.